FXyz-Client/src/main/java/org/fxyz3d/client/SimpleWindowFrame.java

```diff
@@ -213,25 +214,26 @@ private void initWindowControls() {
             stage.setY(e.getScreenY() - mY);
         });
         // window resizing
-        /*southEastResize.setOnMouseEntered(e -> e.consume());
+        southEastResize.setCursor(Cursor.SE_RESIZE);
+
+        southEastResize.setOnMouseEntered(e -> e.consume());
         southEastResize.setOnMouseExited(e -> e.consume());
         southEastResize.setOnMousePressed((e) -> {
             dragOffsetX = (getTranslateX() + getWidth() - e.getScreenX());
             dragOffsetY = (getTranslateY() + getHeight() - e.getScreenY());
-            //e.consume();
         });        
         southEastResize.setOnMouseDragged((e) -> {
-            double x = e.getScreenX() + dragOffsetX,
-                    y = e.getScreenY() + dragOffsetY;
+            double x = e.getScreenX() + dragOffsetX;
+            double y = e.getScreenY() + dragOffsetY;
             double w = x - getTranslateX();
             double h = y - getTranslateY();
 
             setPrefWidth(Math.max(stageMinWidth, w));
```

Comment:
The initial scene is set to 1200x768, but the actual min values are set to 1280x800, so resizing only works to larger values. 

It might be better to set min values like 800x600?

Reply:
Good point. Will do that. Should I add resize areas at the bottom and the right sides?

Reply:
I guess SE is enough for now, but bottom and right will work too.

Reply:
Ok might add S and E at a later time. Updated min values. Should be good to go.